### ISSUE TITLE
Unable to log in to automa.site

### ISSUE THREAD

**Issue by qin9smile** [AutomaApp/automa#1872]
Description of the Problem
I am unable to log in to the automa.site website. Here’s what happens:

Clicking the Login button redirects me to a page where I can choose either GitHub or Google to authenticate.
After completing the authentication process and being redirected back to automa.site, I am still not logged in. The site shows no login information.
I have tried multiple times to log in, but the issue persists.

Steps to Reproduce
Go to https://automa.site/.
Click the Login button in the top-right corner.
Select either GitHub or Google to log in.
After being redirected back to automa.site, observe that the login was unsuccessful.

**Comment by bingren-yd** [comment]
The login service has been restored. I'm sorry for causing problems.

**Comment by qin9smile** [comment]
Last night, I was able to log in to [automa.site](https://www.automa.site/) briefly, but I still couldn't share workflows. Just now, I tried to log in again, and the login page keeps redirecting in a loop, making it impossible to log in. This issue has reoccurred.

Steps to Reproduce

Visit https://www.automa.site/.
Enter login credentials and attempt to log in.
After logging in, the workflow sharing feature does not work properly.
When trying to log in again, the page keeps redirecting to the login page and login is unsuccessful.
